cmd/internal/shared/gatewayconfigurationserver/config.go

```diff
@@ -0,0 +1,27 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shared
+
+import (
+	"go.thethings.network/lorawan-stack/pkg/gatewayconfigurationserver"
+)
+
+// DefaultGatewayConfigurationServerConfig is the default configuration for the Application Server.
+var DefaultGatewayConfigurationServerConfig = gatewayconfigurationserver.Config{}
```

Comment:
I think we should `RequireAuth: true` here. Both the basicstation and The Things Gateway can do auth just fine.